src/components/PostPageDetail/index.js

```diff
@@ -213,14 +207,6 @@ const PostPageDetailIdComponent = (props) => {
     }
   };
 
-  const onComment = (isAnonimity, anonimityData) => {
-    if (typeComment === 'parent') {
-      commentParent(isAnonimity, anonimityData);
-      if (props?.refreshParent) {
-        props.refreshParent();
-      }
-    }
-  };
   const commentParent = async (isAnonimity, anonimityData) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onComment` function has been moved to a different part of the code and now only calls `commentParent` if `typeComment === 'parent'`. This change seems to be part of a larger refactor of the commenting functionality. Ensure that all instances where `onComment` is used have been updated to handle this change.
```diff
- const onComment = (isAnonimity, anonimityData) => {
-   if (typeComment === 'parent') {
-     commentParent(isAnonimity, anonimityData);
-     if (props?.refreshParent) {
-       props.refreshParent();
-     }
-   }
- };
+ const commentParent = async (isAnonimity, anonimityData) => {
+   setLoadingPost(true);
+   try {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->